components/builder-web/stylesheets/_mixins.scss

```diff
@@ -21,11 +21,12 @@
 
 @mixin chevron-right {
   position: absolute;
-  width: 20px;
-  height: 20px;
-  right: 0;
-  top: 50%;
+  right: $small-spacing;
+  height: rem(24);
+  width: rem(24);
+  top: 46%;
```

Comment:
I haven't been able to actually look at this yet, but I'd expect the `top 50%`+`translate -50%` trick to keep things vertically centered across the board. Did that not seem to happen with the size change?